xxh3.h

```diff
@@ -902,32 +907,39 @@ XXH_PUBLIC_API XXH128_hash_t
 XXH3_128bits_withSeed(const void* data, size_t len, XXH64_hash_t seed)
 {
     if (len <= 16) return XXH3_len_0to16_128b(data, len, seed);
+    if (len > 128) return XXH3_hashLong_128b(data, len, seed);
```

Comment:
It seems this new order adds a branch for any `len > 16`, thus affecting length in the 16-32 range,
which happen to be my primary target at work.
On benchmark, it seems I can measure a small performance degradation in such range, but I need more time to test, as there is ample opportunity for measurement noise.